manager/integration/tests/test_basic.py

```diff
@@ -2496,6 +2496,11 @@ def test_expansion_canceling(client, core_api, volume_name, pod):  # NOQA
     11. Validate the volume content, then check if data writing looks fine
     12. Clean up pod, PVC, and PV
     """
+    static_sc_name = "longhorn"
+    setting = client.by_id_setting(SETTING_DEFAULT_LONGHORN_STATIC_SC)
+    setting = client.update(setting, value=static_sc_name)
+    assert setting.value == static_sc_name
```

Comment:
We already decided to deprecate this setting. There is no need to care this setting.
Same below

Reply:
changed